File: bootpay/src/main/java/kr/co/bootpay/IScriptFuction.java
package kr.co.bootpay;

import android.webkit.JavascriptInterface;

public interface IScriptFuction {
    @JavascriptInterface
    void error(String data);

    @JavascriptInterface
    void cancel(String data);

    @JavascriptInterface
    void confirm(String data);

    @JavascriptInterface
    void done(String data);
}

File: bootpay/src/main/java/kr/co/bootpay/IScriptFuction.kt
package kr.co.bootpay

import android.webkit.JavascriptInterface

interface IScriptFuction {
    @JavascriptInterface
    fun error(data: String)

    @JavascriptInterface
    fun close(data: String)

    @JavascriptInterface
    fun cancel(data: String)

    @JavascriptInterface
    fun ready(data: String)

    @JavascriptInterface
    fun confirm(data: String)

    @JavascriptInterface
    fun done(data: String)
}
